content/docs/hooks-faq.md

```diff
@@ -463,51 +462,51 @@ function Parent({ a, b }) {
 }
 ```
 
-Note that this approach won't work in a loop because Hook calls [can't](/docs/hooks-rules.html) be placed inside loops. But you can extract a separate component for the list item, and call `useMemo` there.
+Ten en cuenta que este método no funcionará en un ciclo porque las llamadas a Hooks [no pueden](/docs/hooks-rules.html) ser puestas dentro de ciclos. Pero puedes extraer un componente separado para el item de la lista, y llamar `useMemo` allí.
 
-### How to create expensive objects lazily? {#how-to-create-expensive-objects-lazily}
+### ¿Cómo crear objetos costosos de manera perezosa (lazy)? {#how-to-create-expensive-objects-lazily}
```

Comment:
Idk how will be the right way to translate this but it looks pretty rare. I'd put `¿Cómo crear objetos pesados de manera perezosa/floja (lazy)?`

Reply:
El término preferido en el glosario para «*lazy initialization*» es «inicialización diferida». Por tanto, por consistencia, probablemente lo mejor sería traducirlo de la siguiente forma: ¿Cómo crear objetos costosos de manera diferida?

Reply:
Yeah, much better. // Si, está mucho mejor.

Reply:
```suggestion
### ¿Cómo crear objetos costosos de manera diferida (lazy)? {#how-to-create-expensive-objects-lazily}
```

Reply:
Will change, this also applies in some other places, Will change those also for consistency